pipenv/utils/funktools.py

```diff
@@ -71,8 +71,7 @@ def unnest(elem: Iterable) -> Any:
         for el in target:
             if isinstance(el, Iterable) and not isinstance(el, str):
                 el, el_copy = tee(el, 2)
-                for sub in unnest(el_copy):
-                    yield sub
+                yield from unnest(el_copy)
```

Comment:
I actually have a bigger refactor/simplification of this method in my open PR that will conflict with this.

Reply:
Well. Then let's merge this with out caring for ruff. We'll fix ruff in the next PRs